services/quiz/src/pages/StudyDetailPage/components/QuizItem/index.tsx

```diff
@@ -29,86 +22,82 @@ type QuizItemColorMap = {
 };
 
 const QUIZ_ITEM_COLOR: QuizItemColorMap = {
-  default: {
+  DONE: {
     button: 'general',
     timer: 'gray060',
   },
-  disabled: {
+  TO_DO: {
     button: 'primary',
     timer: 'primary',
   },
-  activated: {
+  IN_PROGRESS: {
     button: 'primary',
     timer: 'primary',
   },
 };
 
-// TODO: 아이콘들 Image로 되어있는것 추후 Icon 컴포넌트로 변경해야 함
 // TODO: 카운트 돌아가고 있을시 첫 렌더링에 반영되도록 해야함.
 // TODO: 퀴즈 추가 버튼 1초 늦게 나오는 문제 해결해야 함,,,
-export function QuizItem({
-  index,
-  weekNumber,
-  title,
-  openDate,
-  limitTime,
-  creator,
-  absentee,
-  setAddQuizState,
-}: QuizItemProps) {
-  const limitDate = getLimitDate(openDate, limitTime);
-  const [quizItemType, setQuizItemType] = useState(getQuizItemType(openDate, limitDate) as QuizStatus);
-  const [timer, setTimer] = useState(quizItemType === 'default' ? '00:00:00' : limitTime);
-  const [isFold, setIsFold] = useState(quizItemType !== 'default');
+export function QuizItem({ round, quiz }: QuizItemProps) {
+  const limitDate = new Date(quiz.endedAt);
```

Comment:
```suggestion
export function QuizItem({ round, quiz }: QuizItemProps) {
 const { quizStatus, ... } = quiz;
  const limitDate = new Date(quiz.endedAt);
```

구조 분해 할당해서 쓰면 좋을 것 같아요 중복이 많네요

Reply:
넵넵~!